### ISSUE TITLE
Internal inaccurate package dependencies cause error

### ISSUE THREAD

**Issue by MR4online** [twurple/twurple#353]
### Code

Our  package.json looks like this:

```json
    "@twurple/api": "5.0.17",
    "@twurple/auth": "5.0.17",
```

So we force loading version 5.0.17

### Expected behavior

Version 5.0.17 is used by the api and auth package.

### Actual Behavior

The api and auth packages are loading 5.0.19 which causes an Typescript Error.

node_modules/@twurple/api/package.json:
```json
    "@twurple/api-call": "^5.0.17",
    "@twurple/common": "^5.0.17",
```
-> here v. **5.0.19** is installed, where incompatible changes were made.

### Typescript Error

```
node_modules/@twurple/common/lib/errors/CustomError.d.ts(3,44): error TS2304: Cannot find name 'ErrorOptions'.
node_modules/@twurple/auth/lib/errors/InvalidTokenError.d.ts(7,27): error TS2304: Cannot find name 'ErrorOptions'.
node_modules/@twurple/api/lib/errors/StreamNotLiveError.d.ts(7,27): error TS2304: Cannot find name 'ErrorOptions'.
```

**Comment by Desnoo** [comment]
As a temporary fix we use the following configs:

We have set the following 2 packages to our last known working version to 5.0.17 inside our package.json.
```
"resolutions": {
  "@twurple/common": "5.0.17",
  "@twurple/api-call": "5.0.17"
},
"dependencies": {
    "@twurple/api": "5.0.17",
    "@twurple/auth": "5.0.17",
    ....
}
```
and also added the preinstall hook to our package.json
```
"preinstall": "npx npm-force-resolutions",
```

**Comment by d-fischer** [comment]
A better workaround for this would be to include `es2022.error` in the `lib` option in your tsconfig.

I'll fix this in the upcoming days, it's my bad, I forgot that my use of newer libs in my own tsconfig kinda "bleeds down" to consumers who also use TS.

**Comment by d-fischer** [comment]
The TS error should be fixed in version 5.1.1. Please update.

Also, the version mismatch is not what caused this error. A proper version management is on my to-do list. lerna does not have all features I need for this.

**Comment by MR4online** [comment]
thanks a lot!
